packages/admin/src/index.ts

```diff
@@ -75,6 +75,7 @@ export { FinalFormSwitch } from "./form/Switch";
 export { FinalFormSearchTextField, FinalFormSearchTextFieldProps } from "./form/FinalFormSearchTextField";
 export { Select } from "./form/Select";
 export { FinalFormSelect } from "./form/FinalFormSelect";
+export { FinalFormMultiSelect, FinalFormMultiSelectClassKey } from "./form/FinalFormMultiSelect";
```

Comment:
I think it makes sense to export the props here as well:

```suggestion
export { FinalFormMultiSelect, FinalFormMultiSelectClassKey, FinalFormMultiSelectProps } from "./form/FinalFormMultiSelect";
```

Reply:
thats right :) i will export those props too :)